ArduPlane/GCS_Mavlink.cpp

```diff
@@ -166,7 +166,7 @@ void GCS_MAVLINK_Plane::send_nav_controller_output() const
             targets.y * 1.0e-2f,
             targets.z * 1.0e-2f,
             wp_nav_valid ? quadplane.wp_nav->get_wp_bearing_to_destination() : 0,
-            wp_nav_valid ? MIN(quadplane.wp_nav->get_wp_distance_to_destination(), UINT16_MAX) : 0,
+            wp_nav_valid ? MIN(quadplane.wp_nav->get_wp_distance_to_destination() * 1.0e-2f, UINT16_MAX) : 0,
```

Comment:
I've noticed this before and forgot to submit it. We are trying to not add any more `f` and instead leave it to the compiler. 

```suggestion
            wp_nav_valid ? MIN(quadplane.wp_nav->get_wp_distance_to_destination() * 1.0e-2, UINT16_MAX) : 0,
```

Reply:
I would vote for: `* 0.01` its easier to understand at a glance and is shorter.

Reply:
I absolutely agree, I hated the scientific notation, and the "f", but did it this way to match the lines immediately above and below.

Should I make a separate pull request that fixes those?

Reply:
Just chuck it all in this PR.

Reply:
Agree! I even did a quick grep of the scientific notation and found it everywhere so I was like, meh.